samcli/local/init/templates/cookiecutter-aws-sam-hello-powershell/README.md

```diff
@@ -0,0 +1,40 @@
+# Cookiecutter SAM for PowerShell based Lambda functions
```

Comment:
In parallel, we're currently simplifying the root README for each template to we don't confuse customers when they go to contribute and end up reading the wrong file (cookiecutter-aws-sam-hello-powershell/README vs cookiecutter-aws-sam-hello-powershell/{{cookiecutter.project_name}}/README).

Could you update this file based on this reference for .NET?

https://github.com/awslabs/aws-sam-cli/blob/0622b9bb9b601a6c8c1e4054c2a0c93c2e70c96b/samcli/local/init/templates/cookiecutter-aws-sam-hello-dotnet/README.md